modules/services/agentless-scanning/variables.tf

```diff
@@ -67,4 +67,10 @@ variable "stackset_admin_role_arn" {
   description = "(Optional) stackset admin role to run SELF_MANAGED stackset"
   type        = string
   default     = ""
-}
\ No newline at end of file
+}
+
+variable "main_region_agentless_role_arn" {
```

Comment:
I would suggest renaming this variable to `primary_region_agentless_role_arn`. Reason being, to keep it consistent with what the customer sees on the UI, and `Primary AWS Region` is the field they are more familiar with.

Reply:
I would suggest not having this variable. I do not understand the technical reason for its necessity. eventbridge has this same problem, but resolves it internally within the module without introducing new input parameters, which is what should done here. If this is not possible, we should articulate the reasons why.

Reply:
done